app/[id]+api.ts

```diff
@@ -12,16 +16,22 @@ export function GET(request: Request, { id }: { id: string }) {
   return Response.json(guest);
 }
 
-export function DELETE(request: Request, { id }: { id: string }) {
-  console.log('delete guest', id);
-
-  const guests = deleteGuest(Number(id));
+export async function DELETE(
+  request: Request,
+  { id }: { id: string },
+): Promise<Response> {
+  const guests = await deleteGuest({
+    id: Number(id),
+  });
 
   return Response.json(guests);
```

Comment:
this is inconsistent to the other `GET /guests` API route, where we return an object:

```ts
  return Response.json(
    { guests: guests },
```

let's also return an object with the key `guests` here, and the guests inside

Reply:
fixed, and return an object with key `guest` [a2fde60](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/a2fde607a42aa392b94c3612b4662b8c5eb572f5)